src/transformStateWithClones.js

```diff
@@ -7,7 +7,29 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
-}
+  let currentState = { ...state };
```

Comment:
The variable name `currentState` is good, but it's even better to use a name like `stateCopy` to make it clear that this is a copy of the state at each step, as per the checklist item 5.